torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -587,7 +597,36 @@ void ComputeSubgraphInMKLDNN(Node* subgraph_node) {
     }
 
     if (body_node->kind() == aten::relu6) {
-      body_node->replaceWithNewSymbol(prim::MKLDNNRelu6);
+      WithInsertPoint insert_guard{body_node};
+      auto out_node = body_node->owningGraph()->create(
+          {prim::MKLDNNHardTanh}, {body_node->input(0)}, 1);
+      out_node->f_(attr::min_val, 0.);
```

Comment:
Can you factor this and the `aten::hardtanh` handling below into a common helper ?